digitaltwin_client/samples/digitaltwin_sample_environmental_sensor/digitaltwin_sample_environmental_sensor.c

```diff
@@ -603,25 +608,37 @@ DIGITALTWIN_CLIENT_RESULT DigitalTwinSampleEnvironmentalSensor_SendTelemetryMess
 
     float currentTemperature = 20.0f + ((float)rand() / RAND_MAX) * 15.0f;
     float currentHumidity = 60.0f + ((float)rand() / RAND_MAX) * 20.0f;
-
+    float currentLongitude = -124.0f + ((float)rand() / RAND_MAX) * 59.0f;
+    float currentLatitude = 24.0f + ((float)rand() / RAND_MAX) * 25.0f;
+    
     char currentTemperatureMessage[128];
     char currentHumidityMessage[128];
+    char currentLocationMessage[128];
 
-    sprintf(currentTemperatureMessage, "%.3f", currentTemperature);
-    sprintf(currentHumidityMessage, "%.3f", currentHumidity);
+    sprintf(currentTemperatureMessage, "{\"%s\":%.3f}", DigitalTwinSampleEnvironmentalSensor_TemperatureTelemetry, currentTemperature);
```

Comment:
>sprintf [](start = 4, length = 7)

These different messages should all be collapsed into one SendTelemetryAsync() call.  The idea of this message change is to have fewer msgs (and bandwidth overhead and user $$ / message) reduced.

I do not know how to express this JSON though -e.g. can it just be {"temp:".., "humid":...} or whether we have to have a top-level named item (e.g. "sensorData": {...}).

Sam or Miguel would know.